src/adapters/authAdapter.js

```diff
@@ -19,6 +19,14 @@ class AuthAdapter {
       client_secret: process.env.REACT_APP_CLIENT_SECRET
     });
   };
+
+  static forgotPassword = (email) => {
+    return Api.post('api/v1/passwords', {
+      user: { email: email },
+      client_id: process.env.REACT_APP_CLIENT_ID,
+      client_secret: process.env.REACT_APP_CLIENT_SECRET
+    });
+  };
```

Comment:
@rafayet-monon  How about refactor this a bit? What do you think?
```suggestion
DEFAULT_PAYLOAD = {
    client_id: process.env.REACT_APP_CLIENT_ID,
    client_secret: process.env.REACT_APP_CLIENT_SECRET
  };
  
  static forgotPassword = (email) => {
    return Api.post('api/v1/passwords', {
      user: { email: email },
      ...this.DEFAULT_PAYLOAD
    });
  };
```